salt/cloud/clouds/vmware.py

```diff
@@ -569,6 +569,34 @@ def _wait_for_task(task, vm_name, task_type, sleep_seconds=1, log_level='debug')
         raise task.info.error
 
 
+def _wait_for_host(host_ref, task_type, sleep_seconds=5, log_level='debug'):
+    time_counter = 0
+    while host_ref.runtime.connectionState != 'notResponding':
+        message = "[ {0} ] Waiting for host {1} to finish [{2} s]".format(host_ref.name, task_type, time_counter)
+        if log_level == 'info':
+            log.info(message)
+        else:
+            log.debug(message)
+        time.sleep(int(sleep_seconds))
```

Comment:
Should a max_retries be added here? This reads to me like it might never break out of the loop if the host doesn't respond.